From marc@redhat.comMon Jun  5 14:46:37 1995
Date: Mon, 5 Jun 1995 14:46:01 -0400 (EDT)
From: Marc Ewing <marc@redhat.com>
To: ewt@redhat.com
Subject: NetKit-B diffs

    [The following text is in the "ISO-8859-1" character set]
    [Your display is set for the "US-ASCII" character set]
    [Some characters may be displayed incorrectly]

diff -uNr NetKit-B-0.06/lpr/common_source/pathnames.h NetKit-B-0.06.new/lpr/common_source/pathnames.h
--- NetKit-B-0.06/lpr/common_source/pathnames.h	Mon May 23 05:05:02 1994
+++ NetKit-B-0.06.new/lpr/common_source/pathnames.h	Wed Sep  7 12:21:29 1994
@@ -36,7 +36,7 @@
 
 #include <paths.h>
 
-#define	_PATH_DEFDEVLP		"/dev/lp"
+#define	_PATH_DEFDEVLP		"/dev/lp0"
 #define	_PATH_DEFSPOOL		"/var/spool/lpd"
 #define	_PATH_HOSTSEQUIV	"/etc/hosts.equiv"
 #define	_PATH_HOSTSLPD		"/etc/hosts.lpd"
diff -uNr NetKit-B-0.06/lpr/lpq/Makefile NetKit-B-0.06.new/lpr/lpq/Makefile
--- NetKit-B-0.06/lpr/lpq/Makefile	Mon May 23 05:05:02 1994
+++ NetKit-B-0.06.new/lpr/lpq/Makefile	Wed Sep  7 13:02:22 1994
@@ -3,7 +3,8 @@
 lpq:	lpq.o
 
 install: lpq
-	install -s -m6555 lpq /usr/bin
+	install -s -g lp -m6555 lpq /usr/bin
+	isntall lpq.1 /usr/man/man1
 
 clean:
 	rm -f lpq *.o
diff -uNr NetKit-B-0.06/lpr/lpr/Makefile NetKit-B-0.06.new/lpr/lpr/Makefile
--- NetKit-B-0.06/lpr/lpr/Makefile	Mon May 23 05:05:02 1994
+++ NetKit-B-0.06.new/lpr/lpr/Makefile	Wed Sep  7 11:32:31 1994
@@ -3,7 +3,7 @@
 lpr:	lpr.o
 
 install: lpr
-	install -s -m6555 lpr /usr/bin
+	install -s -g lp -m6555 lpr /usr/bin
 	install -m644 lpr.1 /usr/man/man1
 
 clean:
diff -uNr NetKit-B-0.06/lpr/lprm/Makefile NetKit-B-0.06.new/lpr/lprm/Makefile
--- NetKit-B-0.06/lpr/lprm/Makefile	Mon May 23 05:05:02 1994
+++ NetKit-B-0.06.new/lpr/lprm/Makefile	Wed Sep  7 11:32:49 1994
@@ -3,7 +3,7 @@
 lprm:	lprm.o
 
 install: lprm
-	install -s -m6555 lprm /usr/bin
+	install -s -g lp -m6555 lprm /usr/bin
 	install -m644 lprm.1 /usr/man/man1
 
 clean:
